pkg/volume/flexvolume/probe.go

```diff
@@ -213,8 +209,8 @@ func (prober *flexVolumeProber) initWatcher() error {
 
 // Creates the plugin directory, if it doesn't already exist.
 func (prober *flexVolumeProber) createPluginDir() error {
-	if _, err := os.Stat(prober.pluginDir); os.IsNotExist(err) {
-		err := os.MkdirAll(prober.pluginDir, 0755)
+	if _, err := prober.fs.Stat(prober.pluginDir); os.IsNotExist(err) {
```

Comment:
Log a warning & event from here that Flex volume plugin directory is not present. This is not supposed to happen.

Reply:
@derekwaynecarr is this an appropriate place for an event or should they just log an error? IIRC events are really more appropriate for alerting on state changes, and it's probably better to just log an error here. Though I suppose you could consider "Recreating" a state change...